tests/onboarding/test_onboarding_syncing.py

```diff
@@ -74,7 +74,7 @@ def test_sync_device_during_onboarding(multiple_instance, user_data):
 
         with step('Sign in to synced account'):
             sync_result.sign_in()
-            SplashScreen().wait_until_appears().wait_until_hidden()
+            SplashScreen().wait_until_hidden()
```

Comment:
The splash screen should appear first, and we need to wait for it, because `SplashScreen().wait_until_hidden()` executes before it appears.

Reply:
oh, got it, you are waiting for it here: `sign_in`